src/shapes/sprite.js

```diff
@@ -14,16 +14,18 @@ function SpriteShape(params) {
     dx: params.x,
     dy: params.y,
     speed: properties.speed,
-    animation: null
+    update: null,
+    animation: properties.animation === true ? true : false,
```

Comment:
But not is better normalize input for **bool**? If give a **string** like param, my variable will is `true` but it's not a **bool**.

```js
var properties = {};
properties.animation = "some-random-string"; // typeof string, but true in condition.
if (properties.animation) {
   console.log("properties.animation is true");
}
```